docs/ref/contrib/admin/index.txt

```diff
@@ -853,159 +854,11 @@ subclass::
-
-          Also as a convenience, the ``ModelAdmin`` object is passed to
-          the ``lookups`` method, for example if you want to base the
-          lookups on the available data::
-
-              class AdvancedDecadeBornListFilter(DecadeBornListFilter):
-
-                  def lookups(self, request, model_admin):
-                      """
-                      Only show the lookups if there actually is
-                      anyone born in the corresponding decades.
-                      """
-                      qs = model_admin.get_queryset(request)
-                      if qs.filter(birthday__gte=date(1980, 1, 1),
-                                    birthday__lte=date(1989, 12, 31)).exists():
-                          yield ('80s', _('in the eighties'))
-                      if qs.filter(birthday__gte=date(1990, 1, 1),
-                                    birthday__lte=date(1999, 12, 31)).exists():
-                          yield ('90s', _('in the nineties'))
-
-    * a tuple, where the first element is a field name and the second
-      element is a class inheriting from
-      ``django.contrib.admin.FieldListFilter``, for example::
-
-          class PersonAdmin(admin.ModelAdmin):
-              list_filter = (
-                  ('is_staff', admin.BooleanFieldListFilter),
-              )
-
-      You can limit the choices of a related model to the objects involved in
-      that relation using ``RelatedOnlyFieldListFilter``::
-
-          class BookAdmin(admin.ModelAdmin):
-              list_filter = (
-                  ('author', admin.RelatedOnlyFieldListFilter),
-              )
-
-      Assuming ``author`` is a ``ForeignKey`` to a ``User`` model, this will
-      limit the ``list_filter`` choices to the users who have written a book
-      instead of listing all users.
-
-      You can filter empty values using ``EmptyFieldListFilter``, which can
-      filter on both empty strings and nulls, depending on what the field
-      allows to store::
-
-          class BookAdmin(admin.ModelAdmin):
-              list_filter = (
-                  ('title', admin.EmptyFieldListFilter),
-              )
-
-      .. note::
-
-        The ``FieldListFilter`` API is considered internal and might be
-        changed.
-
-      .. note::
-
-        The :class:`~django.contrib.contenttypes.fields.GenericForeignKey`
-        field is not supported.
-
-    List filter's typically appear only if the filter has more than one choice.
-    A filter's ``has_output()`` method controls whether or not it appears.
-
-    It is possible to specify a custom template for rendering a list filter::
-
-        class FilterWithCustomTemplate(admin.SimpleListFilter):
-            template = "custom_template.html"
+    list page of the admin.
 
-    See the default template provided by Django (``admin/filter.html``) for
-    a concrete example.
+    At it's simplest ``list_filter`` takes an iterable of field names to
```

Comment:
an `iterable` of field names is used here, but in the moved document, it says ` a list
or tuple of elements` ... which is more precise? In an ideal world, it'd be any iterable (or at least a sequence?), but there's a boatload of places in Django that specifically do `isinstance(x, (list, tuple))` - is this one of them (e.g; via checks framework or runtime, or?)

Reply:
I suspect any sequence would do, but let's stick with `a list or tuple` for now.